docs/diagrams/DeleteSequenceDiagram.puml

```diff
@@ -3,7 +3,7 @@
 
 box Logic LOGIC_COLOR_T1
 participant ":LogicManager" as LogicManager LOGIC_COLOR
-participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":NutriGoalsParser" as NutriGoalsParser LOGIC_COLOR
```

Comment:
I believe the `DeleteSequenceDiagram.png` is not yet updated? :laughing:

Reply:
Fixed!